server/src/main/java/org/opentosca/toscana/plugins/cloudformation/util/MappingUtils.java

```diff
@@ -8,21 +8,58 @@
 
 import com.google.common.collect.ImmutableList;
 
+/**
+ Utility class for mapping values.
+
+ @see CapabilityMapper */
 public class MappingUtils {
+
+    /**
+     Gets a {@link List} of memory size which correspond with {@code numCpus}.
+     <br>
+     The memory integers are pulled from every {@link org.opentosca.toscana.plugins.cloudformation.mapper.CapabilityMapper.InstanceType}
+     , from the {@code instanceTypes} {@link ImmutableList}, that also has {@code numCpus} as cpu value.
+
+     @param numCpus       the numCpus to find the corresponding memory values for
+     @param instanceTypes the instance types to search in
+     @return a {@link List} of valid memory values that correspond with {@code numCpus}
+     */
     public static List<Integer> getMemByCpu(Integer numCpus, ImmutableList<CapabilityMapper.InstanceType> instanceTypes) {
         return instanceTypes.stream()
             .filter(u -> u.getNumCpus().equals(numCpus))
             .map(CapabilityMapper.InstanceType::getMemSize)
             .collect(Collectors.toList());
     }
 
+    /**
+     Gets a {@link List} of numbers of cpus which correspond with {@code memSize}.
+     <br>
+     The numbers of cpu integers are pulled from every {@link org.opentosca.toscana.plugins.cloudformation.mapper.CapabilityMapper.InstanceType}
+     , from the {@code instanceTypes} {@link ImmutableList}, that also has {@code memSize} as memory value.
+
+     @param memSize       the memSize to find the corresponding number of cpus for
+     @param instanceTypes the instance types to search in
+     @return a {@link List} of numCpu values that correspond with {@code memSize}
+     */
     public static List<Integer> getCpuByMem(Integer memSize, ImmutableList<CapabilityMapper.InstanceType> instanceTypes) {
         return instanceTypes.stream()
             .filter(u -> u.getMemSize().equals(memSize))
             .map(CapabilityMapper.InstanceType::getNumCpus)
             .collect(Collectors.toList());
     }
 
+    /**
+     Gets the {@link String} representation of an {@link org.opentosca.toscana.plugins.cloudformation.mapper.CapabilityMapper.InstanceType}
```

Comment:
Don't overuse links. While they are useful, they make reading the source code more difficult. Linking to `String` does not make any sense. Every developer knows what you mean by that. At most, use `{@code String}`.

Btw you can also import the `InstanceType`.  This would also increase the readability.